tests/db_functions/migrations/0002_create_test_models.py

```diff
@@ -55,8 +55,8 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='DecimalModel',
             fields=[
-                ('n1', models.DecimalField(decimal_places=2, max_digits=6)),
-                ('n2', models.DecimalField(decimal_places=2, max_digits=6)),
+                ('n1', models.DecimalField(decimal_places=2, max_digits=6, null=True, blank=True)),
```

Comment:
`null=True, blank=True` are not necessary for this patch. Please revert them:
```suggestion
                ('n1', models.DecimalField(decimal_places=2, max_digits=6)),
```